CMakeModules/build_freetype.cmake

```diff
@@ -8,7 +8,7 @@ IF (${CMAKE_BUILD_TYPE} STREQUAL "Debug")
 ENDIF()
 
 SET(freetype_location
-    ${prefix}/lib/${CMAKE_STATIC_LIBRARY_PREFIX}freetype${LIB_POSTFIX}${CMAKE_STATIC_LIBRARY_SUFFIX})
+    ${prefix}/lib/${CMAKE_SHARED_LIBRARY_PREFIX}freetype${LIB_POSTFIX}${CMAKE_SHARED_LIBRARY_SUFFIX})
```

Comment:
Is there a reason why this is being changed to shared library ? freetype v2.7.1 doesn't support MSVC toolchain on Windows, however static builds are still possible. Did you test this change on Windows with VS2015 ?